app/src/main/java/org/literacyapp/authentication/TrainingHelper.java

```diff
@@ -54,29 +66,35 @@ public TrainingHelper(Context context){
         studentImageDao = daoSession.getStudentImageDao();
         studentImageFeatureDao = daoSession.getStudentImageFeatureDao();
         studentImageCollectionEventDao = daoSession.getStudentImageCollectionEventDao();
-        svm = new SupportVectorMachine(context, Recognition.TRAINING);
+        studentDao = daoSession.getStudentDao();
+        svmTrainingFile = new File(AiHelper.getSvmDirectory(), "training");
```

Comment:
Ok. I was just mentioning it because I mistaked it by being a folder because of the name. But I guess it's clear just by looking at the filename variable anyway.